Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -346,11 +361,12 @@ def fetch_indicators(client: Client,
                      limit: int = -1,
                      is_fetch: bool = True) -> List[Dict]:
     params_dict = clean_user_query(query) if query else build_params_dict(tags, attribute_type)
-
     if limit and limit not in params_dict:
         params_dict['limit'] = limit
-
-    response = client.search_query(params_dict)
+    response = search_query_indicators_pagination(client, params_dict) if is_fetch else client.search_query(params_dict)
+    demisto.debug(f'fetch_indicators response: {response}')
```

Comment:
We debug the response multiple times, lets consider removing some of the irrelevant debug logs.